### ISSUE TITLE
Global search

### ISSUE THREAD

**Issue by MartinMuzatko** [hackmdio/codimd#1080]
I want to be able to find pads by other users tagged with a certain tag. in search, I can only find my own pads, but not the pads I have already found or other pads that are public.

Is there a way?

**Comment by ccoenen** [comment]
Right now, there is no way to do this. We've been recently discussing a very similar topic in our chat room. I'd like to extend an invitation to [join us there](https://riot.im/app/#/room/#codimd:matrix.org).

The discussion spun up around a profile page we might introduce down the line. And whether or not a user's pads should be listed there or not. In the end consensus was, that this should be strictly opt-in. Given the large community CodiMD has, and the different ways it is used, I would like to discuss these changes and how it affects different people. I know that some people rely on documents being "hidden".

In short: we currently have no way of telling which notes would be considered "public" and which ones would be considered "private".

**Comment by ccoenen** [comment]
Yes, we're open to that. To my knowledge, nobody is working on this. It has been discussed a few times, though: https://github.com/hackmdio/codimd/issues/12 https://github.com/hackmdio/codimd/issues/109 https://github.com/hackmdio/codimd/issues/166 Please continue discussion in one of these issues or create a specialised ticket for your request if they don't match your use cases.
